backtesting/results.py

```diff
@@ -107,13 +107,13 @@ def show(instances: typing.List['CoinInsights'], currency_symbol: str):
             stats.append([c.pair,
                         c.n_trades, 
                         round(c.market_change, 2),
-                        round(c.cum_profit_percentage / c.n_trades, 2),
+                        round(c.cum_profit_percentage / (c.n_trades if c.n_trades > 0 else 1), 2),
```

Comment:
As mentioned before, I would change the 'else 1' to 'else 0' here.

Reply:
@WillemGerritzen could you change this?

Reply:
Hi Guys, just joining in. I just wanted to draw your attention to this cool github feature.  
```suggestion
                        round((c.cum_profit_percentage / c.n_trades) if c.n_trades > 0 else 0, 2),
```